docs/agents/sematext-agent/autodisco/credential-sets.md

```diff
@@ -0,0 +1,8 @@
+title: Defining credential sets
+description:  Defining credential sets
+
+When running in Kubernetes **Sematext Agent** fetches the credentials from user-defined secrets. Typically, you create one secret per service type, such as `st-redis-creds`. However, if you're deploying various clusters of Redis services and you want to assign each cluster a different
```

Comment:
"various clusters of Redis services" -> "various clusters of Redis services in the same Kubernetes cluster".